### PR TITLE
Add content from Cindy's doc into challenge page

### PR DESCRIPTION
**Pull request by atn34**
This required rendering markdown for parts of the challenge page so that
we can link things from the text. Collapsing the habit overview stuff
does not work yet.

### CODE REVIEW

File: src/components/challenge/ChallengeSection.js [hunk 1]
```diff
@@ -12,7 +14,7 @@ class ChallengeSection extends Component {
               <span className="point-count">{this.props.challenge.points}</span>
             </div>
           </div>
-          <div className="challenge-card-body">{this.props.challenge.text}</div>
+          <div className="challenge-card-body">{<QuokkaMarkdown source={this.props.challenge.text}/>}</div>
```
Comment (whittlbc) [inline src/components/challenge/ChallengeSection.js]: Can probs get rid of the extra pair of curly braces between the `div` and the `QuokkaMarkdown`.

File: src/components/challenge/OverviewSection.js [hunk 2]
```diff
@@ -1,14 +1,34 @@
...
+                    </div>
+                  </div>
+                  )
+                )}
+              </div>
+            </div>
+          </div>
         </div>
-      </div>
+      </div >
```
Comment (whittlbc) [inline src/components/challenge/OverviewSection.js]: extra space

**Comment by whittlbc** [comment]
Left a couple styling change requests but otherwise, looks good to go.

File: src/components/challenge/ChallengeSection.js (hunk 1 above)
Comment (atn34) [reply src/components/challenge/ChallengeSection.js]: Done.

File: src/components/challenge/OverviewSection.js (hunk 2 above)
Comment (atn34) [reply src/components/challenge/OverviewSection.js]: Done.

**Comment by atn34** [comment]
Closed by 2e4629c

**Pull request closed without merging**